### ISSUE TITLE
Display new facet `Taxonomic Rank` in Object tab. 

### ISSUE THREAD

**Issue by roamye** [project-lux/lux-frontend#404]
**Problem Description:**
YPM has new meta-classifications for additional faceting. Rather than displaying these new meta-classifications under `Categorized As` they would like a separate facet to be created

**Expected Behavior/Solution:**
Create new facet `Taxonomic Rank` and display all new YPM meta-classifications. 

**Requirements:**
- New facet `Taxonomic Rank` will be displayed on Object tabs. 
- Must display under `Categorized As` (TBD/Need to check) 
- Facet must contain all new YPM meta-classifications. 
- YPM meta-classifications must not appear under `Categorized As`. 

**Needed for promotion:**
If an item on the list is not needed, it should be crossed off but not removed.

- [ ] Wireframe/Mockup - Mike
- [ ] Committee discussions - Sarah
- [ ] Feasibility/Team discussion - Sarah
- [ ] Backend requirements - TBD
- [ ] Frontend requirements- TBD
- [ ] Are new regression tests required for QA - Amy
- [ ] Questions
- List of questions for discussions. Answers should be documented within the issue.

**UAT/LUX Examples:**

- https://lux-front-sbx.collections.yale.edu/view/results/objects?q=%7B%22AND%22%3A%5B%7B%22text%22%3A%22bombus%22%2C%22_lang%22%3A%22en%22%7D%2C%7B%22text%22%3A%22new%22%2C%22_lang%22%3A%22en%22%7D%2C%7B%22text%22%3A%22haven%22%2C%22_lang%22%3A%22en%22%7D%2C%7B%22text%22%3A%22remington%22%2C%22_lang%22%3A%22en%22%7D%2C%7B%22text%22%3A%221982%22%2C%22_lang%22%3A%22en%22%7D%2C%7B%22text%22%3A%22Westville%22%2C%22_lang%22%3A%22en%22%7D%5D%7D&sq=bombus+new+haven+remington+1982+Westville.

**Dependencies/Blocks:**

- Blocked By: 

**Related Github Issues:**
N/A

**Related links:**
Bugherds: https://www.bugherd.com/projects/284041/tasks/2605

**Wireframe/Mockup:**
Current: 
![image](https://github.com/user-attachments/assets/9aad5827-a38e-4f95-8e25-bdaee1261555)

Expected:

**Comment by roamye** [comment]
Per the UX/BA 10/16 meeting @miketullo95 will need some example options to create the mockup.

**Comment by prowns** [comment]
@kkdavis14 - on https://lux-front-tst.collections.yale.edu/view/results/objects?q=%7B%22text%22%3A%22horseshoe%22%2C%22_lang%22%3A%22en%22%7D&sq=horseshoe

Is this how the facets would be divided? 

Categorized As
- three-dimensional(14157)
- Fossils(13689)
- hypotype(472)
- Biological specimens(398)
- Books(127)
- archival materials(44)
- Numismatics(24)

Taxonomic Rank
- Eurypterus(3584)
- Eurypterida(2788)
- Eurypterus remipes(1984)
- Hughmilleria socialis(765)
- Eurypterus lacustris(738)
- Bellinurus danae(537)
- Xiphosurida(446)
- Synziphosurina(340)
- Erieopterus(302)
- Eurypterus pittsfordensis(272)
- Hughmilleria(243)
- Pterygotus (Acutiramus) macrophthalmus(148)
- Limulus polyphemus(145)
- Tremaglaspis(107)
- Paracarcinosoma scorpionis(103)
- Pterygotidae(100)
- Waeringopterus apfeli(86)
- Eurypterus remipes lacustris(83)
- Acutiramus cummingsi(73)
- Erieopterus microphthalmus(70)
- Hutchinsoniella macracantha(60)
- Dolichopterus macrocheirus(52)
- Dolichopterus(51)
- Eurypterus dekayi(51)
- Dolichopterus jewetti(47)
- Aglaspida(38)
- Eurypterus remipes remipes(37)
- Pseudoniscus(35)
- Carcinosomatidae(35)
- Erettopterus osiliensis(34)
- Stylonuracea(29)
- Erettopterus(23)

**Comment by kkdavis14** [comment]
@prowns lgtm!

**Comment by prowns** [comment]
@kkdavis14 - Will that be in the data or does front-end need to do something?  Trying to identify example for @miketullo95  to use in the mockup.

**Comment by kkdavis14** [comment]
@prowns the data constants are in the data now, yes. each taxon is classified as something, which has a meta-classification/data constant of a wikidata.

**Comment by prowns** [comment]
For TF - need to clarify what is expected from YPM on this facet and what the label ought to be.   Is it set of classifications pulled out if they are taxonomic? If so, this has nothing to do with the data constants.

**Comment by roamye** [comment]
@prowns - should this be a topic for next weeks agenda? Unsure who is able to clarify what is needed from YPM, would that be @kkdavis14 ?

**Comment by prowns** [comment]
@roamye - I am checking with YPM.

**Comment by clarkepeterf** [comment]
@roamye @prowns 

Generally we create Advanced Search terms alongside facets. This is not a hard requirement, but if we would like and Advanced Search term I think we should do both at the same time.

I have questions about this:

1. There will just be one 'Scientific Name' field, correct? We are not going to have separate facets for 'Species', 'Genus', 'Order', etc?
2. Do we want 'Categorized As' to still include Scientific Names in its list of categories? Or should 'Categorized As' be changed to exclude Scientific Names?

**Comment by kkdavis14** [comment]
@clarkepeterf the answer to the second question is (with 99% certainty) exclude Scientific Names from Categorized As--as the point of this work was to de-clutter this facet :)

**Comment by clarkepeterf** [comment]
@kkdavis14 sounds good.

Further questions here would be:
- Do we want to also make the Categorized As search term in Advanced Search exclude Scientific Name?
- Do we want to modify Categorized As for People & Groups to have similar exlusions (Categorized As would exclude Nationality, Gender, Occupation)

**Comment by kkdavis14** [comment]
@clarkepeterf These are good questions. 
I would think yes to the first question, but no the second, which is inconsistent. Maybe a conversation to bring to a larger group for input?

**Comment by prowns** [comment]
@kkdavis14 - are you expecting that biological taxa, such as Kingdom, Species, Order, etc, appear as a separate facet as well?  Or will those not be offered to users as a facet option?  

cc: @clarkepeterf and @roamye

**Comment by kkdavis14** [comment]
@prowns no, it seems from Gary's comment in Teams YPM is looking for one facet--"Scientific Name". I'm only expecting whatever their requirement is :)

**Comment by clarkepeterf** [comment]
I am changing the name of this ticket.

From:
"Display new facet Scientific Name in Object tab of Simple and Advanced Search"

To:
"Display new facet Scientific Name in Object tab"

Reason:
facets always display in both Simple and Advanced Search, we don't need to call them out in the title.

**Comment by roamye** [comment]
@miketullo95 - two wireframes are needed on this issue. Please see: [this comment](https://github.com/project-lux/lux-frontend/issues/404#issuecomment-2436163357) for details on available options within the `Scientific Name` facet.
